internal/metrics/collectors/manager.go

```diff
@@ -63,8 +71,14 @@ func NewLocalManagerMetricsCollector(constLabels map[string]string) *LocalManage
 }
 
 // IncNginxReloadCount increments the counter of successful NGINX reloads and sets the last reload status to true
-func (nc *LocalManagerMetricsCollector) IncNginxReloadCount() {
-	nc.reloadsTotal.Inc()
+func (nc *LocalManagerMetricsCollector) IncNginxReloadCount(isEndPointUpdate bool) {
+	if isEndPointUpdate {
+		nc.reloadsTotal.WithLabelValues("endpoints").Inc()
+		nc.reloadsTotal.WithLabelValues("other")
```

Comment:
could you do these initilizations only once in `NewLocalManagerMetricsCollector` ?

```
nc.reloadsTotal.WithLabelValues("other")
nc.reloadsTotal.WithLabelValues("endpoints")
```